src/amo/sagas/collections.js

```diff
@@ -213,25 +229,77 @@ export function* updateCollection({
       `/${lang}/${clientApp}/collections/${user}/${effectiveSlug}/`
     ));
 
+    const slugWasEdited = slug && slug !== collectionSlug;
```

Comment:
I know we don't really need this line, but trying to inline it into the `if` statement below made the code a lot less understandable, so I kept this.